pkg/pgmodel/end_to_end_tests/query_integration_test.go

```diff
@@ -939,13 +1316,16 @@ func generateSamples(index int) []prompb.Sample {
 	var (
 		delta     float64 = float64(index * 2)
 		timeDelta int64   = 30000
+
+		// Jan 1st, 2020
+		startTime int64 = 1577836800000
```

Comment:
nit: let's make this a constant used here and in startTime on line 667 too? Maybe end time as well (and use here)